wetterdienst/api.py

```diff
@@ -200,7 +200,7 @@ def collect_data(self) -> Generator[pd.DataFrame, None, None]:
                             create_new_file_index=False,
                         )
                     except InvalidParameterCombination:
-                        log.info(
+                        log.warning(
```

Comment:
It makes me sad that codecov is complaining here and will fail the integration. However, I definitively see the point of retroactively improving test coverage by using this mechanism.